src/js/main.js

```diff
@@ -0,0 +1,85 @@
+import { openPost, showTitleBody, showUserEmail } from "./info-modal.js";
+import { getSearchResults, displaySearchResults } from "./search.js";
+
+// Variable and listener for searchbar functions
+const searchBtn = document.getElementById('headerSearchBtn');
+searchBtn.addEventListener('click', function () {
+  getSearchResults()
+  displaySearchResults();
+});
+
+// Fetch posts from api for posts, return as .json data, and pass to displayPosts function.
+async function getPostData() {
+  try {
+    const response = await fetch("https://jsonplaceholder.typicode.com/posts/");
+    const postData = response.json();
+    return postData;
+  } catch(error) {
+    alert('Error Data');
+  }
+}
+
+// Call function to define data for window onload
+async function manageData () {
+  const data = await getPostData();
+  displayPosts(data)
+}
+
+window.onload = manageData;
+
+// Get Comments from json server
+async function getComments () {
+  try {
```

Comment:
Where is the finally ?? Bro

try {
  try_statements
}
catch (exception_var) {
  catch_statements
}
finally {
  finally_statements
}

Reply:
All perfect. Good work!